schemas/cs3/tbl/t_item/item.ksy

```diff
@@ -0,0 +1,109 @@
+meta:
+  id: item
+  endian: le
+  encoding: UTF-8
+  imports:
+    - /components/cs3/character
+    - /components/cs3/item_type
+    - /components/cs3/effect
+    - /components/common/element
+seq:
+  - id: item_id
+    type: u2
+  - id: character_restriction
+    type: character
+  - id: flags
+    type: strz
+  - id: item_type
+    type: item_type
+  - id: item_subtype
+    type: u2 # TODO: make an enum for this?
+
+  - id: element
+    type: u1 # can't use element enum here because it's a u2
+
+  - id: slash
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: thrust
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: pierce
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: strike
+    type: u1
+    doc: |
+      Number of stars, from 0 to 4 (inclusive)
+  - id: unknown_byte
+    type: u1
+    doc: |
+      Only nonzero for (some) weapons.
+  - id: target_type
+    type: u1
+    doc: |
+      For weapons and consumables, this is the target type.
+      However, this field is also nonzero for a few other items (such as Eternal Kandelaar and costumes)
+  - id: range
+    type: f4
+  - id: area
+    type: u1
+
+  - id: unknown_float_1
+    type: f4
+  - id: unknown_float_2
+    type: f4
+  - id: unknown_float_3
+    type: f4
+
+  - id: effects
+    type: effect
+    repeat: expr
+    repeat-expr: 5
+
+  - id: str
+    type: s2
+  - id: df
+    type: s2
+    doc: |
+      Defense modifier.
+      (Not called "def" because that's a python keyword)
```

Comment:
I think I just use the full name for all the stats to not run into this

Reply:
Makes sense, I'll do that

Reply:
I changed most of them, but left ATS/ADF/MOV as is. I think that should be ok?